src/LodeRunner.Core/Models/ClientStatus.cs

```diff
@@ -119,6 +120,7 @@ public LoadClient LoadClient
         /// <value>
         /// The TTL.
         /// </value>
+        [Required]
         public int Ttl
```

Comment:
Should the Required attribute also be added to LastUpdated, LastStatusChange, Status, and LoadClient (based on Data Dictionary table)

Reply:
Thank you for catching this! Just updated!